product_security/red_hat_responsibilities/dedicated_network_security.adoc

```diff
@@ -0,0 +1,98 @@
+[[dedicated-network-security]]
+= Network Security
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+[[network-segregation]]
+== Network Segregation
+
+{product-title} uses a software-defined networking (SDN) approach to provide a
+unified cluster network that enables communication between pods across the
+cluster. This pod network is established and maintained by the OpenShift SDN,
+which configures an overlay network using Open vSwitch (OVS).
+
+{product-title} uses the *ovs-networkpolicy* plug-in, which allows project
+administrators to configure their own isolation policies using NetworkPolicy
+objects. These policies can be fully managed by project owners and
+`dedicated-admin` users to add network-level isolation for the projects.
+
```

Comment:
Add following below:
Additionally, some dedicated clusters also support the ovs-multitenant plug-in that provides project-level isolation for pods and services. Each project receives a unique Virtual Network ID (VNID) that identifies traffic from pods assigned to the project. By default, Pods from different projects cannot send packets to or receive packets from pods and services of a different project. However, the cluster administrator can join projects and later isolate them using admin commands. For more information on Pod network management see https://docs.openshift.com/dedicated/admin_guide/managing_networking.html